README.md

```diff
@@ -1,2 +1,11 @@
 # docs.reana.io
 REANA user documentation web site
```

Comment:
Please add `## About` here, also readme and license badges, so that we have a nice harmony with `www.reana.io`

Reply:
Updated.